### PR TITLE
Preparing for Storage release

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.6.0-preview.1 (Unreleased)
+## 12.6.0 (20202-08-31)
```
Comment (kasobol-msft) [inline sdk/storage/Azure.Storage.Blobs/CHANGELOG.md:3]: should this be 12.5.2?

File: sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.4.0-preview.1 (Unreleased)
+## 12.4.0 (2020-08-31)
```
Comment (kasobol-msft) [inline sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md:3]: should this be 12.3.2?

File: sdk/storage/Azure.Storage.Blobs/CHANGELOG.md (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/CHANGELOG.md:3]: We have an interface change (grouped `BlobBaseClient.OpenRead()` optional parameters into `BlobOpenReadOptions`), so we have a minor version bump.

File: sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md:3]: We have an interface change (grouped `ShareFileClient.OpenRead()` optional parameters into `ShareFileOpenReadOptions`), so we have a minor version bump.

File: sdk/storage/Azure.Storage.Blobs/CHANGELOG.md (hunk 1 above)
Comment (amnguye) [reply sdk/storage/Azure.Storage.Blobs/CHANGELOG.md:3]: There's an extra 2 after 2020

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs/CHANGELOG.md (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/CHANGELOG.md]: Fixed, thanks!

**Review by amnguye — APPROVED** [review]

**Review by tg-msft — APPROVED** [review]

**Pull request merged**